pom.xml

```diff
@@ -99,47 +114,9 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-
-    <!-- WIRED TEST RUNNER DEPENDENCIES -->
```

Comment:
I've replaced all of these dependencies with new dependencies on Bitbucket Server's common test modules (`bitbucket-it-common` and `bitbucket-page-objects`). Those will transitively pull in all the dependencies you need to write browser and REST integration tests.